### PR TITLE
docs: complete the translation of dql/window

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)
some Chinese content haven't been translated to English
the boundary table is outdated

* **What is the new behavior (if this is a feature change)?**
complete translation
updated table
change 'calculation' to 'computation'
change the example title format to

### CODE REVIEW

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md [hunk 1]
```diff
@@ -51,7 +51,9 @@ WindowInstanceNotInWindow
 				:: = 'INSTANCE_NOT_IN_WINDOW'
 ```
 
-*Window call function* implements functionality similar to aggregate functions. The difference is that the window call function does not need to pack the query results into a single line of output—in the query output, each line is separated. However, the window caller can scan all rows that may be part of the current row's group, depending on the grouping specification of the window caller (the `PARTITION BY` column). The syntax for calling a function from a window is one of the following:
+*Window call function* is similar to aggregate functions. The difference is that the window call function does not need to pack the query results into a single line when output the results. Instead, each line is separated when using WINDOW clause.
```
Comment (lumianph) [inline docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: *Window call function*
=>
The *window* function

Comment (lumianph) [reply docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: two occurrences

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md [hunk 2]
```diff
@@ -51,7 +51,9 @@ WindowInstanceNotInWindow
 				:: = 'INSTANCE_NOT_IN_WINDOW'
 ```
 
-*Window call function* implements functionality similar to aggregate functions. The difference is that the window call function does not need to pack the query results into a single line of output—in the query output, each line is separated. However, the window caller can scan all rows that may be part of the current row's group, depending on the grouping specification of the window caller (the `PARTITION BY` column). The syntax for calling a function from a window is one of the following:
+*Window call function* is similar to aggregate functions. The difference is that the window call function does not need to pack the query results into a single line when output the results. Instead, each line is separated when using WINDOW clause. 
+However, the window caller can scan all rows that may be part of the current row's group, depending on the grouping specification of the window caller (the `PARTITION BY` on columns).
```
Comment (lumianph) [inline docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: window caller
=>
window function

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md [hunk 3]
```diff
@@ -77,9 +79,9 @@ SELECT select_expr [,select_expr...], window_function_name(expr) OVER window_nam
 
 ## Boundary Description
```
Comment (lumianph) [inline docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: Description

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md [hunk 4]
```diff
@@ -77,9 +79,9 @@ SELECT select_expr [,select_expr...], window_function_name(expr) OVER window_nam
...
+|:-------------------------------------------------------|--------------|---------------------|---------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| WINDOW Clause                                          | **``✓``**    |                     | **``✓``**           | The window clause is used to define one or several windows. Windows can be named or anonymous. Users can call aggregate functions on the window to perform analysis (```sql agg_func() over window_name```). During Online Serving, please follow [Online Serving下Window的使用规范](../deployment_manage/ONLINE_SERVING_REQUIREMENTS.html#online-servingwindow) |
```
Comment (lumianph) [inline docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: Online Serving下Window的使用规范
is this a Chinese doc?

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md [hunk 5]
```diff
@@ -128,58 +130,63 @@ WindowFrameBound
...
+- `CURRENT ROW` is the row currently being computed.
+- `UNBOUNDED PRECEDING` indicates the upper bound of this window is unlimited.
+- `expr PRECEDING`
+  - When the window is `ROWS` type, `expr` must be a positive integer, which indicates the upper boundary is the `expr`th row before current row.
+  - When the window type is `ROWS_RANGE`,`expr` should be a time interval, like `10s`, `10m`,`10h`, `10d`. The upper bound is the `expr` ahead of the time of current row.
+- By default, OpenMLDB uses closed interval. To change this, you can use keyword `OPEN`.
 
-#### **Example: 有名窗口（Named Window）**
+```{Note}
+Standard SQL also supports `FOLLOWING` boundary, but OpenMLDB doesn't support it currently.
```
Comment (lumianph) [inline docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: `FOLLOWING` boundary =>
the `FOLLOWING` keyword

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md [hunk 6]
```diff
@@ -197,7 +204,7 @@ WINDOW w1 AS (UNION t2, t3 PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10
 
 ![Figure 3: window union two tables](../dql/images/window_union_2_table.png)
 
-#### **Example: Window with union 样本表不进入窗口**
+**3. Window With `UNION` but Exclude Instance Table**
```
Comment (lumianph) [inline docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: but Exclude => Excluding the

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md [hunk 7]
```diff
@@ -206,7 +213,8 @@ WINDOW w1 AS (UNION t2 PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PR
 
 ![Figure 4: window union one table with instance_not_in_window](../dql/images/window_union_1_table_instance_not_in_window.png)
 
-#### **Example: Window with union 列筛选子查询**
+
+**4. Window With `UNION` Composed Of Subquery**
```
Comment (lumianph) [inline docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: Window With `UNION` Containing Subqueries

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md [hunk 8]
```diff
@@ -245,24 +252,24 @@ WINDOW w1 AS (PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PRECEDING A
...
 ```sql
--- ROWS example
 -- desc: window ROWS_RANGE, 前10s到当前条，同时限制窗口条数不超过3条
 SELECT sum(col2) OVER w1 as w1_col2_sum FROM t1
 WINDOW w1 AS (PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PRECEDING AND CURRENT ROW MAXSIZE 3);
 ```
 
 ```{seealso}
-Aggregate functions that can be used in window calculation, refer to [Built-in Functions](../functions_and_operators/Files/udfs_8h.md)
+Aggregate functions that can be used in window computation, refer to [Built-in Functions](../functions_and_operators/Files/udfs_8h.md)
```
Comment (lumianph) [inline docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: Please refer to [Built-in Functions](../functions_and_operators/Files/udfs_8h.md) for aggregate functions that can be used in window computation.

**Review by lumianph — CHANGES_REQUESTED** [review]
please also address the conflicts

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md (hunk 6 above)
Comment (michelle-qinqin) [reply docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: Exclude Instance Table是sql关键词，也需要修改嘛？

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md (hunk 4 above)
Comment (michelle-qinqin) [reply docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: I have forgotten this when checking.
thans for reminding me!

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md (hunk 1 above)
Comment (michelle-qinqin) [reply docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: > two occurrences

sorry, I don't konw what to do with this comment?

**Comment by michelle-qinqin** [comment]
> please also address the conflicts

the conflicts have been solved

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md (hunk 1 above)
Comment (lumianph) [reply docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: > 

sorry, I just meant there were two occurrences of `window call function`

File: docs/en/reference/sql/dql/WINDOW_CLAUSE.md (hunk 6 above)
Comment (lumianph) [reply docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: the keyword is INSTANCE_NOT_IN_WINDOW?

Comment (michelle-qinqin) [reply docs/en/reference/sql/dql/WINDOW_CLAUSE.md]: 是我搞混了，最新提交的已经修改好啦

**Review by lumianph — APPROVED** [review]

**Pull request merged**